sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -742,6 +749,22 @@ class SparkSession private(
     DataType.fromJson(dataTypeString)
   }
 
+  /**
+   * Stops all active streaming queries
+   */
+  private def stopStreamingQueries(): Unit = {
```

Comment:
If we have this function, `StreamingQueryManager` might be a better place instead of here `SparkSession`.

Reply:
I 've moved it with `private[sql]`. I don't want to introduce new APIs in StreamingQueryManager.

Reply:
@dongjoon-hyun , moving this to `StreamingQueryManager` fails in some cases. E.g. if the Spark context was already stopped, the SparkSession still tries to lazily load the `streams` (StreamingQueryManager) to invoke the method and the lazy initialization fails.

So I have moved the method back into `SparkSession`.

Reply:
In that case, companion object will be the right solution. Could you try to make and use `object StreamingQueryManager`? We had better collect the logic into one place.

Also, cc @jose-torres , @tdas , @zsxwing, @cloud-fan  since this is a design issue for `StreamingQueryManager`.